packages/node/src/index.ts

```diff
@@ -528,20 +531,99 @@ export const startDevServer: StartDevServer = async opts => {
     filename: 'package.json',
   });
   const pkg = pathToPkg ? require_(pathToPkg) : {};
+  const tsNodePath = require_.resolve('ts-node');
+  const esmLoader = join(tsNodePath, '..', '..', 'esm.mjs');
```

Comment:
I wonder if it would be better to read the `ts-node` package.json `"bin"` or `"exports` object to retrieve this path. In case it was ever to change in the future we'd have to manually figure it out again.

---

Since I still had it up locally, I quickly looked into what it might be and I think it would be `exports["./esm"]`, which is seemingly no better than the existing hardcoded path. This comment can probably be ignored but I'm leaving the thought here in case others have more ideas.

Reply:
I think its fine as-is. It wont change unless we bump `ts-node`. If the case when a newer version of `ts-node` changed the file format, it will cause the tests to fail.